spec/features/pipelines/pipelineRun_status_test.go

```diff
@@ -0,0 +1,45 @@
+package pipelines
+
+import (
+	. "github.com/onsi/ginkgo"
+	"github.com/openshift-pipelines/release-tests/pkg/helper"
+	"github.com/openshift-pipelines/release-tests/pkg/pipelines"
+)
+
+var _ = Describe("PipelineRun and TaskRun status", func() {
+
+	// Setup up state for each test spec
+	// create new project (not set as active) and new context directory for each test spec
+	// This is before every spec (It)
+	//var once sync.Once
+	BeforeEach(func() {
+
+		clients, ns, cleanup = helper.NewClientSet()
```

Comment:
This supports my ask not to create a namespace in `NewClientSet`... when I run all tests, it would try to create the namespace multiple times.

Reply:
Yes, it is required when I run these features tests in parallel, I should run each test on individually on their own namespace